test/screenshotter/ss_data.yaml

```diff
@@ -132,7 +132,6 @@ KaTeX:
 Kern:
     tex: \frac{a\kern{1em}b}{c}a\kern{1em}b\kern{1ex}c\kern{-0.25em}d
     nolatex: LaTeX fails to typeset this, “Missing number, treated as zero.”
-Lap: ab\mathllap{f}cd\mathrlap{g}hij\mathclap{k}lm \; ab\llap{f}cd\rlap{g}hij\clap{k}lm
```

Comment:
Instead of removing the test completely, could you revert to the updated test you made and then comment it out so that it's easy for people pick up the work?

Reply:
Done